cellprofiler/preferences.py

```diff
@@ -625,6 +626,24 @@ def resources_root_directory():
 directly from the server).\
 """
 
+NORMALIZATION_FACTOR_HELP = """\
+Sets the normalization factor for intensity normalization methods:
+
+-  *{INTENSITY_MODE_LOG}*: Set the gain applied to the each pixel in the image during log normalization. Pixels are
```

Comment:
Can we add a sentence here, similar to the one below, indicating higher values lead to brighter images?  Just for the equation-averse.  Otherwise, I love it!

Reply:
Great idea! I'll add that.